.github/workflows/labeler.yml

```diff
@@ -7,6 +7,9 @@ on:
 env:
   MY_GITHUB_TOKEN: ${{ secrets.APM_TECH_USER_TOKEN }}
 
+permissions:
+  contents: read
```

Comment:
Won't this need permissions to write labels on issues/prs?

Reply:
Arg, you are right. But the auto-merge kicked in.